C#/C#.sublime-syntax

```diff
@@ -1,278 +1,1634 @@
 %YAML 1.2
 ---
 # http://www.sublimetext.com/docs/3/syntax.html
-name: "C#"
+# https://github.com/gwenzek/csharpSyntax
```

Comment:
This should be removed since it isn't really relevant any longer, however you are free to add something like the MIT license here with your name/email.

With a significant rewrite like this, you will likely also get a mention in the changelog, like with your work on Scala.